doc/source/tutorial/assets/america.gml

```diff
@@ -1,4 +1,4 @@
-Creator "igraph version 0.9.3 Sun Nov 14 18:16:32 2021"
+Creator "igraph version 0.10.0-dev+63a49e74 Sun Nov 21 16:02:47 2021"
```

Comment:
It would be better to keep a non-dev version number in any file that is included in the tutorial.

Reply:
Good catch. I'm not actually using this file, so I've removed it for now, but definitely a good thing to keep in mind for the future.